README.md

```diff
@@ -16,7 +16,7 @@ provides an interface to the HDF5 library for the Julia language.
 
 ## Changelog
 
-Please see [HISTORY.jl](HISTORY.md) for the changelog. Most changes have deprecation warnings and thus may not be listed in this file.
+Please see [HISTORY.md](HISTORY.md) for the changelog. Most changes have deprecation warnings and thus may not be listed in this file.
```

Comment:
Let's not touch this file and save the changes for the documentation (it's easy for this to get out of sync).

Reply:
Sure. I was just trying to make it stand out and fix the typo